src/Makefile

```diff
@@ -111,12 +111,12 @@ $(LIBXXHASH):
 $(LIBTRIEMAP):
 	@$(MAKE) -C util/triemap
 
-BLAS_CFLAGS = -fPIC -I $(abs_path)/util -include rmalloc.h -DREDIS_MODULE_TARGET -DGB_MALLOC=rm_malloc -DGB_CALLOC=rm_calloc -DGB_REALLOC=rm_realloc -DGB_FREE=rm_free -DGBCOMPACT
+BLAS_CFLAGS = -fPIC -I $(abs_path)/util -include rmalloc.h -DGB_MALLOC=rm_malloc -DGB_CALLOC=rm_calloc -DGB_REALLOC=rm_realloc -DGB_FREE=rm_free
```

Comment:
Does 3.0 not have a COMPACT flag? Not needing a custom semiring is nice, but I'm surprised the module size didn't increase more.

Reply:
Tim introduced GB_Control.h which gives granular control over what's included in graphblas.a

Reply:
-rwxr-xr-x  1 roilipman  staff   6.7M Jul 12 08:00 ./src/redisgraph.so

Reply:
-rwxr-xr-x  1 roilipman  staff   6.7M Jul 12 08:00 ./src/redisgraph.so